tests/functional/wasm.js

```diff
@@ -28,7 +29,14 @@ function createServer(app) {
 module.exports = {
   async start() {
     const app = express();
-    app.use(express.static(path.join(__dirname, '..', '..', 'wasm')));
+
+    const build_directory = process.env["WASM_BUILD_DIRECTORY"]
+      ? path.parse(process.env["WASM_BUILD_DIRECTORY"])
+      : path.join(__dirname, '..', '..', 'build', 'build');
```

Comment:
path.parse returns Object
path.join returns String, it would be nice if the type is consistent.